manager/service.go

```diff
@@ -116,9 +129,34 @@ func New(cfg qemu.Config, attestationPolicyBinPath string, logger *slog.Logger,
 
 func (ms *managerService) Run(ctx context.Context, c *ComputationRunReq) (string, error) {
 	ms.mu.Lock()
-	cfg := ms.qemuCfg
+	cfg := qemu.VMInfo{
+		Config:    ms.qemuCfg,
+		LaunchTCB: 0,
+	}
 	ms.mu.Unlock()
 
+	cmd := exec.Command("sudo", fmt.Sprintf("%s/attestation_policy", ms.attestationPolicyBinaryPath), "--policy", "196608")
```

Comment:
run conditionally only for when qemu config is set for SEV or SEV_SNP

Reply:
Done